spec/features/cmless_pages_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'support/cmless_markdown_tester'
+
+describe 'Markdown for Cmless classes' do
+
+  @cmless_classes = [About, Collection]
+
+  # Get a list of all markdown files under each Cmless class's ROOT path.
+  @files = @cmless_classes.map{ |klass| Dir["#{klass::ROOT}/**/*.md"] }.flatten
+
+
+  @files.each do |path|
+
+    # Use relative path for easier-to-read output from rspec.
+    relative_path = path.gsub(/^#{File.expand_path('./')}/, '')
```

Comment:
just plain `path.sub(File.expand_path('./'), '')` should work / be more predictable?